src/text-field-base.css.ts

```diff
@@ -0,0 +1,265 @@
+                color: 'transparent',
+                opacity: 0,
+            },
+
+            // Override Chrome input autocomplete styles:
+            '&:-webkit-autofill': {
+                WebkitTextFillColor: vars.colors.textPrimary,
+                // The background can not be overriden, but we can delay the background color transition to avoid the change
+                transitionProperty: 'background-color',
+                transitionDelay: '99999s',
+            },
+        },
+    },
+    commonInputStyles,
+]);
+
+// Firefox: hide value if not valid or focused
+// Only apply when Firefox, otherwise it breaks styles in safari mobile
+export const inputFirefoxStyles = style({
+    selectors: {
+        '&[type="month"]:not(:valid):not(:focus)': {
+            color: 'transparent',
+        },
+        '&[type="date"]:not(:valid):not(:focus)': {
+            color: 'transparent',
+        },
+        '&[type="datetime-local"]:not(:valid):not(:focus)': {
+            color: 'transparent',
+        },
+    },
+});
+
+export const inputWithLabel = style({
+    paddingTop: 28,
+    '@media': {
+        [mq.tabletOrSmaller]: {
+            paddingTop: 24,
+        },
+    },
+    paddingBottom: 8,
+});
+
+export const inputWithoutLabel = style({
+    paddingTop: 16,
+    paddingBottom: 16,
+});
+
+export const endIcon = style([
+    sprinkles({
+        paddingLeft: 8,
+        paddingRight: 16,
+        display: 'flex',
+        alignItems: 'center',
+    }),
+    {
+        alignSelf: 'center',
+    },
+]);
+
+export const startIcon = style([
+    sprinkles({
+        paddingLeft: 12,
+        paddingRight: 12,
+        display: 'flex',
+        alignItems: 'center',
+        height: '100%',
+        position: 'absolute',
+    }),
+    {
+        pointerEvents: 'none', // passthrough click events to the input
+    },
+]);
+
+export const prefix = style([
+    sprinkles({
+        paddingLeft: 12,
+        paddingRight: 16,
+    }),
+    {
+        transition: 'opacity 150ms cubic-bezier(0.0, 0, 0.2, 1) 0ms',
+    },
+]);
+
+export const prefixWithLabel = style([
+    sprinkles({
+        paddingBottom: 8,
+    }),
+    {
+        paddingTop: 28,
+        '@media': {
+            [mq.tabletOrSmaller]: {
+                paddingTop: 24,
+            },
+        },
+    },
+]);
+
+export const prefixWithoutLabel = sprinkles({
+    paddingTop: 16,
+    paddingBottom: 16,
+});
+
+export const menuItem = style([
+    sprinkles({
+        height: 48,
+        display: 'flex',
+        alignItems: 'center',
+        cursor: 'pointer',
+    }),
+    {
+        padding: '6px 16px',
+        transition: 'background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms',
+        selectors: {
+            '&:hover': {
+                background: 'rgba(0, 0, 0, 0.08)',
+            },
+        },
+    },
+]);
+
+export const menuItemSelected = style({
+    background: 'rgba(0, 0, 0, 0.14)',
```

Comment:
this value is replaced in the [Select](https://github.com/Telefonica/mistica-web/pull/598/files#diff-32053d64977f03cbcdefc5b2eaa1d656be9cf26c19894b2a7b13a77e76e4d851) component to `vars.colors.backgroundAlternative` (talked with Yayo)